SCClassLibrary/Common/GUI/PlusGUI/Control/NodeTreeView.sc

```diff
@@ -8,20 +8,28 @@ NodeTreeView {
 		^super.newCopyArgs(server).makeWindow(bounds, parent)
 	}
 
-	makeWindow { |bounds, parent|
+	makeWindow 	{ |bounds, parent|
```

Comment:
Can you remove the whitespace change here? Sorry if that was in the code I posted.

Reply:
Done in https://github.com/supercollider/supercollider/pull/6948/commits/5a271ff11407fb63c63ecf89bab8c9a3e0901415!